framework/build.gradle

```diff
@@ -29,6 +36,14 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.13'
     testImplementation project(':framework-test')
     testImplementation sourceSets.testannotations.output
+
+    // AutoValue support in returns receiver checker
+    implementation "com.google.auto.value:auto-value-annotations:${versions.autoValue}"
```

Comment:
For reviewers: this means the Checker Framework will rely on the AutoValue annotations artifact.  This only contains AutoValue annotations, not any of the AutoValue annotation processors.  We could remove this dependence if desired, but it will require adding some reflection to the checker.

Reply:
We can probably get rid of the Lombok test dependency, sure.

We will get rid of the AutoValue annotations `implementation` as well.  There is a small safety loss, but I think it's worth it to avoid the dependency.